doc/rtd/topics/tutorial.rst

```diff
@@ -87,6 +87,11 @@ We can now verify that cloud-init received the expected user data:
 .. code-block:: shell-session
 
     $ cloud-init query userdata
+
+Which should print the following to the terminal window:
+
+.. code-block::
+
```

Comment:
Line 103 mixes commands and output, and also has an unwanted trailing `$` (not your doing).

Line 118 has a trailing `$` (no your doing).

Reply:
Yeah I wasn't quite sure exactly what to do with this block so I left it alone. I'll fix it up :)